ref/net46/Microsoft.Build.Utilities.Core/Microsoft.Build.Utilities.Core.cs

```diff
@@ -314,7 +314,8 @@ public enum TargetDotNetFrameworkVersion
         Version46 = 7,
         Version461 = 8,
         Version462 = 10,
-        VersionLatest = 10,
+        Version47 = 11,
+        VersionLatest = 11,
```

Comment:
In what way is it breaking? We've been just following the pattern done in the past knowing this needed to be revisited at some point.

We could make VersionLatest a special enum that we always treat as latest? We can't really get rid of it though.